platform_versioned_docs/version-23.2.0/enterprise/advanced-topics/manual-aws-batch-setup.mdx

```diff
@@ -158,25 +160,23 @@ Create a launch template to configure the EC2 instances deployed by Batch jobs:
 
     #cloud-config
     write_files:
-      - path: /root/tower-forge.sh
+      - path: /root/seqera-ce.sh
```

Comment:
This is not a seqera platform managed ce, it would better to not use seqera anywhere

Reply:
@pditommaso it's not Platform-managed, but the resources still relate to a Seqera Platform compute environment. This prefix makes it easier to identify among the rest of users' log groups IMO.

Reply:
My point is that we provide as mere template. is customer problem adapt to their need